src/views/pages/Identity/index.jsx

```diff
@@ -1,11 +1,47 @@
-import React, {useState, useContext} from 'react';
+import React, {useState, useEffect} from 'react';
 import PageTitle from '../../components/PageTitle';
+import Text from '../../components/Text';
+import Button from '../../components/Button';
 import IdentitySubNav from '../../components/IdentitySubNav';
-import {IdentityControls} from './IdentityControls';
-import {useAccount} from '../../hooks/useAccount';
+import {useServiceContext} from '../../../contexts/ServiceContext';
+import TransactionConfirm from '../../../addons/Modal/TransactionConfirm';
 
 function Identity() {
-  const account = useAccount();
```

Comment:
please ensure this line (and the other related lines) remain and don't get blown out with the merge. This is the logic that conditionally renders this based on whether an account is loaded.